### PR TITLE
Simple strings

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

**Review by ErichDonGubler — COMMENTED** [review]
I'm honestly ambivalent about these changes -- I don't consider the gain for changing a single expression to use `Into` a compelling reason to merge, but it _is_ somewhat easier to read that line.

As far as concerns I see, I was initially worried about `to_truncate` actually truncating string values. I'm not super familiar with `TruncatableString`, but it seems like the conversion here is lossless, despite the name. In that case, I think that there's no semantic mismatch with `From`, so that's good?

Also, just to disclaim: in case my reviews haven't been clear: I have no authority in this repo -- just expressing my opinion.

**Comment by djc** [comment]
So my idea was that it would be better, where conversion to string is needed, to rely on the opentelemetry conversions, because it ought to be useful for these to be consistent. The `From` impl is indeed a cosmetic change, where I felt that only a single conversion from `Value` to `AttributeValue` makes sense and a `From` impl is a nice way to make that idiomatically obvious, but I do feel it improves readability at the conversion site.

File: src/lib.rs [hunk 1]
```diff
@@ -278,18 +278,21 @@ impl SpanExporter for StackDriverExporter {
...
-    Value::Array(_) => attribute_value::Value::StringValue(to_truncate(String::from(v))),
-  };
-  AttributeValue { value: Some(new_value) }
+impl From<Value> for AttributeValue {
+  fn from(v: Value) -> AttributeValue {
+    use proto::google::devtools::cloudtrace::v2::attribute_value;
+    AttributeValue {
+      value: Some(match v {
+        Value::Bool(v) => attribute_value::Value::BoolValue(v),
+        Value::Bytes(_) => attribute_value::Value::StringValue(to_truncate(v.into())),
```
Comment (Xaeroxe) [inline src/lib.rs:287]: [This conversion operates on the assumption the underlying data is valid UTF-8](https://docs.rs/opentelemetry/0.8.0/src/opentelemetry/api/core.rs.html#179). That's not an assumption I want to go forward with. I do really like the rest of this PR! So if we could restore the hex encoding I'd be happy to merge it.

Comment (Xaeroxe) [reply src/lib.rs:287]: Hm well seeing the `Bytes` type isn't present in the newer version of `opentelemetry` means I actually don't care about this line anymore as it's just going to get merge conflict resolved out of existence.

**Pull request merged**